src/backend/utils/indexer.js

```diff
@@ -82,13 +82,14 @@ const search = async (textToSearch, filter = 'post') => {
         fields: createFieldsFromFilter(filter),
       },
     },
+    sort: [{ published: { order: 'desc' } }],
   };
 
   const {
     body: { hits },
   } = await client.search({
     from: 0,
-    _source: ['id'],
+    _source: ['id', 'published'],
```

Comment:
Don't think we need the published here, we just need it to be sorted when we get the results which is what line 85 does